src/gl/directx11/shaders/tileFragmentShader.hlsl

```diff
@@ -29,6 +29,8 @@ PS_OUTPUT main(PS_INPUT input)
 
   output.Color0 = float4(col.xyz * input.colour.xyz, input.colour.w);
   
-  output.Color0.a = input.depth.x / input.depth.y; // depth packed here
+  float scale = 1.0 / (u_clipZFar - u_clipZNear);
+  float bias = -(u_clipZNear * 0.5);
+  output.Color0.a = (input.depth.x / input.depth.y) * scale + bias; // depth packed here
```

Comment:
Does this need to be done in the other shaders that pack the depth into the alpha channel?

Reply:
Nope!

This is because tiles don't write their depth explicitly in the fragment shader. Tiles are 'tesselated enough' to be able to write the LogZ depth per vertex, then interpolate. This allows the hardware to perform normal depth optimizations, while still giving us a logarithmic distribution :).